pkg/cmd/pr/create/create.go

```diff
@@ -193,8 +193,8 @@ func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Co
 	fl.StringVarP(&opts.BaseBranch, "base", "B", "", "The `branch` into which you want your code merged")
 	fl.StringVarP(&opts.HeadBranch, "head", "H", "", "The `branch` that contains commits for your pull request (default: current branch)")
 	fl.BoolVarP(&opts.WebMode, "web", "w", false, "Open the web browser to create a pull request")
-	fl.BoolVarP(&opts.Autofill, "fill", "f", false, "Do not prompt for title/body and just use commit info")
-	fl.BoolVar(&opts.FillFirst, "fill-first", false, "Do not prompt for title/body and just use first commit info")
+	fl.BoolVarP(&opts.Autofill, "fill", "f", false, "Do not prompt for title/body and just use commit info (could be overwritten by --title and/or --body)")
+	fl.BoolVar(&opts.FillFirst, "fill-first", false, "Do not prompt for title/body and just use first commit info (could be overwritten by --title and/or --body)")
```

Comment:
@williammartin @peterramaldes : Would it be more readable if we moved the details of `--fill` and `--fill-first` to the `Long` documentation section and rephrased the text for these flags:

```suggestion
	fl.BoolVarP(&opts.Autofill, "fill", "f", false, "Use commit info for title/body; can be overridden by other flags")
	fl.BoolVar(&opts.FillFirst, "fill-first", false, "Use commit info for title/body; can be overridden by other flags")
```

I have 3 nits about using flag text to describe this command's behavior:

1. How long the text will be if more flags apply
2. Negative text is harder to read than stating what it will do
3. The confusion from #3888 would benefit from more documentation in the `Long` section explaining the humanization the command and GitHub.com does by default

Reply:
Yes, I think `Long` would make more sense for an extended description of behaviour. In particular:

> A prompt will also ask for the title and the body of the pull request. Use `--title`
and `--body` to skip this, or use `--fill` to autofill these values from git commits

Looks like it could have some additional text to capture that `title` and `body` will override `fill`.

I don't feel that strongly though. We know we have some [real work to do for `gh pr` ](https://github.com/cli/cli/issues/7575).

I don't love your suggestion of `can be overridden by other flags` that much though, I'd rather be explicit where possible and to reconsider if it became unwieldy as you describe in your first point.

Reply:
> I'd rather be explicit where possible and to reconsider if it became unwieldy as you describe in your first point.

Agreed, I struggled finding the right balance between details in the flag specific text versus example(s) and long description of how much detail on which flags interacted.

Reply:
I appreciate the feedback this will aggregate a lot on my knowledge, thank you very much!

Could you guys take a look again at my new proposal (https://github.com/cli/cli/pull/8080/commits/d4668c599ed02f74ad5fc207d51c8dd253ce3a6c)?